# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/automake/hotfix-nios2.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- automake-1.16.5/lib/config.guess.vanilla	2021-10-31 17:11:28.948939280 +0100
+++ automake-1.16.5/lib/config.guess	2021-10-31 17:12:02.681941686 +0100
@@ -1096,6 +1096,9 @@
     mips64el:Linux:*:*)
 	GUESS=$UNAME_MACHINE-unknown-linux-$LIBC
 	;;
+    nios2:Linux:*:*)
+	GUESS=$UNAME_MACHINE-unknown-linux-$LIBC
+	;;
     openrisc*:Linux:*:*)
 	GUESS=or1k-unknown-linux-$LIBC
 	;;
